src/main/java/org/jenkinsci/backend/ircbot/IrcListener.java

```diff
@@ -578,6 +604,89 @@ private boolean createComponent(Channel channel, User sender, String subcomponen
         return result;
     }
 
+    String createUploadPermissionPR(Channel channel, User sender, String jiraIssue, String forkTo, List<String> ghUsers, List<String> releaseUsers) {
+        String prUrl = "";
+        boolean isPlugin = forkTo.endsWith("-plugin");
+        OutputChannel out = channel.send();
+        if(isPlugin) {
+            String branchName = forkTo + "-permissions";
```

Comment:
Would "namespace" it as `ircbot/…` to prevent weird original PR conflicts.

Reply:
Will do